src/test-reader/test-parser.js

```diff
@@ -113,6 +125,18 @@ class TestParser extends EventEmitter {
             treeBuilder.addTestFilter(test => grep.test(test.fullTitle()));
         }
 
+        if (config.lastFailed && config.lastFailed.only) {
+            const failedStringified = this.#failedTests.map(data => JSON.stringify(data));
+            treeBuilder.addTestFilter(test =>
+                failedStringified.includes(
+                    JSON.stringify({
```

Comment:
Why you stringify it twice? To my mind we don't need `JSON.stringify` here and 3 lines above at all.

Reply:
I don't like solution with iterate through all tests in order to find desired one. I described the above solution with `Map`, which should be much faster. What do you think?

It will also allow us to get rid of `json.strigify` here.